dnfdaemon-server/services/repo/repo.cpp

```diff
@@ -120,7 +120,7 @@ dnfdaemon::KeyValueMap repo_to_map(
                 std::vector<std::string> reponames = {libdnf_repo->get_id()};
                 query.ifilter_repoid(reponames);
                 for (auto pkg : query) {
-                    size += pkg.get_download_size();
+                    size += pkg.get_package_size();
```

Comment:
The commit also contains renames of methods, but it is not mentioned in commit message. Please may I ask you to fix it?